_source/_includes/blog_post.html

```diff
@@ -54,23 +54,30 @@ <h1 class="BlogPost-title p-name">
             {%- if page.layout == 'blog_post' and post.changelog -%}
             {% assign last_updated_date = post.changelog[0] | split: '=>' | first | date: "%B %-d, %Y" %}
             <span class="vertical-spacer md-show"></span>
-            <div class="BlogPost-last-updated">
+            <div class="BlogPost-last-updated no-excerpt">
                 Last Updated:<a href="#changelog">&nbsp;{{ last_updated_date }}</a>
             </div>
             <span class="vertical-spacer md-hide"></span>
             {% endif %}
           </div>
-          <div>
+          <div class="BlogPost-extra-byline">
             <!-- time to read -->
             <div class="BlogPost-readtime">
               {% include reading-time.html content=post.content %}
-           </div>
-           <!-- social share -->
-           <div class="BlogPost-social-share">
-              <span class="vertical-spacer"></span>
+            </div>
+            {%- if post.github -%}
+            <!-- Sample repo -->
+            <span class="vertical-spacer no-excerpt"></span>
+            <div class="BlogPost-sample-repo no-excerpt">
+              <a href="{{ post.github }}" target="_blank" title="Sample code on GitHub"><i class="fa fa-github"></i> REPO</a>
```

Comment:
I wonder if CODE would be easier for readers to understand?

Reply:
Code looks fine as well